packages/create-calypso-config/src/index.ts

```diff
@@ -68,8 +68,21 @@ const config =
  */
 const isEnabled =
 	( data: ConfigData ) =>
-	( feature: string ): boolean =>
-		( data.features && !! data.features[ feature ] ) || false;
+	( feature: string ): boolean => {
+		// Feature flags activated from environment variables.
+		if (
+			process.env.ACTIVE_FEATURE_FLAGS &&
+			typeof process.env.ACTIVE_FEATURE_FLAGS === 'string'
+		) {
+			const env_active_feature_flags = process.env.ACTIVE_FEATURE_FLAGS?.split( ',' );
```

Comment:
Hi @renatho, it seems that the usage of `process.env` is causing issues with the Blaze Dashboard ([app/blaze-dashboard](https://github.com/Automattic/wp-calypso/tree/trunk/apps/blaze-dashboard)) because the process object isn't available in the browser environment. 
<img width="527" alt="console" src="https://github.com/Automattic/wp-calypso/assets/9039613/0676cf81-95b0-4cd8-a8c2-37bc707f8bc3">

<img width="568" alt="source" src="https://github.com/Automattic/wp-calypso/assets/9039613/dbb42ad2-f411-4cd8-b278-e95d1765656b">

I noticed that some parts of the codebase already use Webpack configuration to inject `process.env `values. 
https://github.com/Automattic/wp-calypso/blob/6f685104e90a5ba196e48c29fda6153264d4ec93/client/webpack.config.js#L299-L303
Probably something similar needs to be done here or perhaps we need to take some action on our end. Please let me know your thoughts. Thanks!

Reply:
No worries at all @renatho. 
Thanks a lot for looking at it this quickly! Appreciate it. I dropped a comment on the PR.

Reply:
Hey folks 👋🏼 Just noting that the change is also causing issue with the Global Styles Limited modal not loading:

![Markup on 2024-06-24 at 11:58:43](https://github.com/Automattic/wp-calypso/assets/25105483/2e859a2b-ede4-4a65-a7fb-64fbf13e2253)

Regardless, #92029 with the @Alameen688's [suggestion](https://github.com/Automattic/wp-calypso/pull/92029/files#r1649493545 ) applied fixes it

Reply:
I'm still seeing an error in `wpcom-global-styles` when loading a P2:

<img width="1288" alt="Screenshot 2024-06-25 at 11 20 48" src="https://github.com/Automattic/wp-calypso/assets/664258/494ac02d-d8a2-41cd-ac87-85948bf448b8">

Can you deploy the latest build of ETK to wpcom so that the fix gets applied there?

Reply:
Oh! For sure! I'll do it today!

Reply:
@jsnajdr, deploy done, and I just confirmed that the error is not happening anymore. 😉
Thank you for the report!

Reply:
Thank you, I'm confirming that it's now also fixed for me :+1: